uefi-test-runner/src/boot/memory.rs

```diff
@@ -113,9 +113,9 @@ fn memory_map(bt: &BootServices) {
 
     #[cfg(target_arch = "x86_64")]
     {
-        let phys_start = first_desc.phys_start;
+        let phys_start = first_desc.raw.PhysicalStart;
         assert_eq!(phys_start, 0, "Memory does not start at address 0");
     }
-    let page_count = first_desc.page_count;
+    let page_count = first_desc.raw.NumberOfPages;
```

Comment:
Could we use some getter methods here, instead of directly accessing the `raw` field?

Reply:
Indeed!